providers/flagsmith/src/main/java/dev/openfeature/contrib/providers/flagsmith/exceptions/InvalidCacheOptionsExceptions.java

```diff
@@ -0,0 +1,18 @@
+package dev.openfeature.contrib.providers.flagsmith.exceptions;
+
+import dev.openfeature.sdk.ErrorCode;
+import lombok.Getter;
+
+/**
+ * InvalidOptions is the super Exception used when we have a configuration exception.
```

Comment:
Perhaps needs to be updated to reflect the correct class name?

Reply:
Done